HousingSearchApi/V1/Controllers/GetTenureListController.cs

```diff
@@ -0,0 +1,65 @@
+using Amazon.Lambda.Core;
+using Hackney.Core.Logging;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Boundary.Response;
+using HousingSearchApi.V1.Boundary.Responses;
+using HousingSearchApi.V1.Boundary.Responses.Metadata;
+using HousingSearchApi.V1.UseCase.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace HousingSearchApi.V1.Controllers
+{
+    [ApiVersion("1")]
+    [Produces("application/json")]
+    [Route("api/v1/search/tenures")]
+    public class GetTenureListController : BaseController
+    {
+        private readonly IGetTenureListUseCase _getTenureListUseCase;
+
+        public GetTenureListController(IGetTenureListUseCase getTenureListUseCase)
+        {
+            _getTenureListUseCase = getTenureListUseCase;
+        }
+
+        [ProducesResponseType(typeof(APIResponse<GetPersonListResponse>), 200)]
+        [ProducesResponseType(typeof(APIResponse<NotFoundException>), 404)]
+        [ProducesResponseType(typeof(APIResponse<BadRequestException>), 400)]
+        [HttpGet, MapToApiVersion("1")]
+        [LogCall(LogLevel.Information)]
+        public async Task<IActionResult> GetPersonList([FromQuery] GetTenureListRequest request)
+        {
+            if (!ModelState.IsValid)
+            {
+                var errors = new List<ValidationError>();
+
+                var err = new ValidationError();
+
+                err.FieldName = "Insufficient characters";
+                errors.Add(err);
+
+                return new BadRequestObjectResult(new ErrorResponse(errors)
+                {
+                    StatusCode = 400
+                });
+            }
```

Comment:
Could this bit not be done with FluentValidation? Also you are assuming that any invalid model is due to insufficient characters when it might be something else.

Reply:
This is exactly the same as the original search (copy/paste and renamed). It's been there for some time. If it's ok I'll do it on the ticket I open to add tests.